app/javascript/message_format/zh_TW.js.erb

```diff
@@ -0,0 +1 @@
+<%= JsLocaleHelper.output_MF("zh_TW") %>
```

Comment:
I wonder if we should get the name of the locale from the filename? 🤔

Reply:
Yes we could probably improve those files :slightly_smiling_face:

Reply:
Do we even need those files? Could we implement a generic solution that works for all languages? :thinking:

Reply:
We need them at some point, yes, as they’re imported in JS :sweat_smile: I guess we could find a way of generating them on the fly, but we’d need to cache them somehow and make them work with the import-map.

Reply:
Done in https://github.com/discourse/discourse/pull/27570/commits/7b0d7d6a6def5d415bdaa682d524ef721de2dd16 :grin: